.github/workflows/ci-master.yml

```diff
@@ -1,15 +1,20 @@
 name: Continues Integration (Master)
 
 on:
-  workflow_dispatch:
   push:
-    - master
-    - main
-    - "Test-Automation"
+    branches:
+      - "master"
+      - "main"
```

Comment:
Why do you use both master and main branches?

Reply:
because currently it is "master" but I assume we will change it in the future